README.md

```diff
@@ -78,8 +78,12 @@
   * Module: `code_challenges/code_wars/vector_class.py`
   * Tests: `tests/code_wars/test_vector_class.py`
   * Run Time: n/a
+### Football Cards (6kyu)
+  * Module: `code_challenges/code_wars/football_cards`
+  * Tests: `tests/code_wars/test_football_cards`
+  * Run Time: O(n log n)
```

Comment:
What lead you to this Big O choice?

Reply:
i cant remember. i was looking up how to figure it out and something i read made me go this route. after re reading i am thinking it is O(n). there is one for loop and im not doing any sorting.